fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -353,14 +369,14 @@ void FixedLagSmoother::processQueue(fuse_core::Transaction& transaction)
     {
       // The motion model processing failed.
       // Check the transaction timeout to determine if it should be removed or skipped.
-      if (element.transaction->stamp() + params_.transaction_timeout < current_time)
+      if (min_stamp + params_.transaction_timeout < current_time)
```

Comment:
I feel like this should use the max stamp instead. With the assumption the transaction arrived at the smoother at maxStamp() or later.
```suggestion
      if (element.maxStamp() + params_.transaction_timeout < current_time)
```